src/room.html

```diff
@@ -100,15 +113,33 @@
             wasd-to-analog2d
             character-controller="pivot: #player-camera"
             ik-root
+            app-mode-toggle-playing__character-controller="mode: hud; invert: true;"
+            app-mode-toggle-playing__wasd-to-analog2d="mode: hud; invert: true;"
             animated-robot-hands
         >
+
+            <a-entity
+                id="player-hud"
+                hud-controller="head: #player-camera;"
+                vr-mode-toggle-visibility
+                vr-mode-toggle-playing__hud-controller
+            >
+                <a-entity in-world-hud="haptic:#player-right-controller;raycaster:#player-right-controller;" rotation="-39.01000608674482 0 0">
```

Comment:
`-39.01000608674482` do we really need that precision?

Reply:
:) I think this probably came from the inspector, will just round it off heh

Reply:
✔️